### PR TITLE
feat: Add listIsActive method

### PR DESCRIPTION
**Pull request by anirudhbs**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,6 +80,8 @@
     "prosemirror-schema-basic": "^1.1.2",
     "prosemirror-schema-list": "^1.1.6",
     "prosemirror-state": "^1.3.4",
+    "prosemirror-tables": "^1.2.5",
```
Comment (mcwinter07) [inline package.json]: Can't see where this is being used, is this a necessary dependency?

**Review by mcwinter07 — COMMENTED** [review]
Just a small query on the Prosemirror tables package that's in there, might have been left in from an experiment, but doesn't look like a dependency? Other than that It looks good to me, assuming there'll be a PR in the @kaizen/rich-text-editor following this?

File: package.json (hunk 1 above)
Comment (anirudhbs) [reply package.json]: `prosemirror-utils` 0.9.6 requires `prosemirror-tables`. I have now upgraded `prosemirror-utils` to 1.0.0-0, which doesn't need the dependency anymore

**Comment by anirudhbs** [comment]
You're right @mcwinter07. Changes in the RTE component will be done after this is merged

**Review by mcwinter07 — APPROVED** [review]
Looks all good in my books 👍  thanks for updating that dependency

**Pull request merged**